app/lib/irs1040_schedule_eic_pdf.rb

```diff
@@ -0,0 +1,43 @@
+class Irs1040ScheduleEicPdf
```

Comment:
@caitbot Seems like we're using EITC and EIC interchangably which also is what the IRS does. Wondering if it makes sense to use one or the other consistently within the code base, or if reflecting what the IRS calls things at different places makes more sense.